src/traversals/metric_travelling_salesman.jl

```diff
@@ -0,0 +1,73 @@
+"""
+	dfs_short_circuit(g, s)
+
+`g` is a spanning tree of some graph.
+Converts the spanning tree into a tour starting at `s`by
+performing depth first search on it but avoids traversing a node more than once.
+The tour is represented by a permutation of the vertices of the graph. 
+"""
+function dfs_short_circuit(g::AbstractGraph, s::T) where T <: Integer
+
+	nvg = nv(g)
+    seen = zeros(Bool, nvg)
```

Comment:
bitvector?

Reply:
That's strange, in my text editor, the indentation seems fine.